routers/api/v1/org/org.go

```diff
@@ -295,7 +295,7 @@ func Get(ctx *context.APIContext) {
 	//   "200":
 	//     "$ref": "#/responses/Organization"
 
-	if !models.HasOrgOrUserVisible(ctx.Org.Organization.AsUser(), ctx.Doer) {
+	if !organization.HasOrgOrUserVisible(db.DefaultContext, ctx.Org.Organization.AsUser(), ctx.Doer) {
```

Comment:
General question, can't we use the `ctx`(*context.APIContext) that's being passed from http?

Reply:
yes and we should

Reply:
Then consider this a lot of nitpicks, because it got me quieted confused while reading the code. Because there's quite a few situations where such `ctx` then can be used.